requirements/development.txt

```diff
@@ -3,6 +3,7 @@
 
 # Useful debugging tools
 django-debug-toolbar<2.0
+django-extensions<2.2
```

Comment:
This is already in https://github.com/readthedocs/ethical-ad-server/blob/master/requirements/base.txt.